lib/funky/page.rb

```diff
@@ -29,10 +29,10 @@ def self.find(page_id)
     #
     # @return [Array<Funky::Video>] multiple Funky::Video objects containing data
     #   fetched by Facebook Graph API.
-    def videos
-      videos = Funky::Connection::API.fetch(
-        "#{id}/videos?fields=id,title,description,created_time,length,comments.limit(0).summary(true),likes.limit(0).summary(true),reactions.limit(0).summary(true)",
-        is_array: true)
+    def videos(options = {})
+      path_query = "#{id}/videos?fields=id,title,description,created_time,length,comments.limit(0).summary(true),likes.limit(0).summary(true),reactions.limit(0).summary(true)"
+      path_query << "&since=#{options[:since]}" if options[:since]
+      videos = Funky::Connection::API.fetch_all(path_query)
```

Comment:
I used `fetch_all` instead of `fetch` method for token-based pagination. I thought time-based pagination (`fetch` with `is_array: true`) can be hard to work with since or until parameter